src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+public class Ui {
+
+    enum Keywords {
+        mark,
+        unmark,
+        delete,
+        todo,
+        deadline,
+        event
+    }
+
+    /**
+     * A constructor to initialize the Ui Object.
+     */
+    public Ui() {}
+
+    public static String straightLine = "  ----------------------------------------------------------------------------------";
+
+    public String logo  =         "___   ___    ___  ____                     ___   ___ \n"
+                               + "|  |  /  /  |  |  \\  \\      ------      /  /   |  | \n"
+                               + "|   /   /   |  |   \\  \\    / / \\ \\    /  /    |  | \n"
+                               + "|   \\  \\   |  |     \\  \\  /  /   \\ \\  /  /     |  | \n"
+                               + "|    \\  \\  |  |      \\  \\/  /     \\ \\/  /      |  | \n"
+                               + "|___| \\__\\ |__|       \\____/       \\____/       |__| \n";
+
+    /**
+     * Output the greeting message when starting up the project.
+     */
+    public void printGreeting() {
+        System.out.println("\nHello from\n" + logo);
+
+        System.out.println("Hello! I'm KiwiQE :) \nWhat can I do for you? \n");
+    }
+
+    /**
+     * Output the goodbye message when terminating the project.
+     */
+    public void printGoodbyeMessage() {
+        System.out.println(straightLine + "\n  sayonara, goodbye\n" + straightLine);
+    }
+
+    /**
+     * Output the task list.
+     */
+    public void printTaskListEmpty() {
+        System.out.println(straightLine + "\n" + "  Nothing to do currently ehe\n" + straightLine);
+    }
+
+    /**
+     * Output the corresponding customised Index Out Of Bounds Exception message.
+     *
+     * @param keyword A keyword object to identify which message to output.
+     */
+    public void printIndexOutOfBoundsException(Keywords keyword) {
+        switch (keyword) {
+        case delete:
+            System.out.println(straightLine + "\n  Can't delete something that isn't there...\n"
+                    + straightLine + "\n");
+            break;
+        case unmark:
+            System.out.println(straightLine + "\n  HEY THERE'S NO SUCH TASK! >:(\n"
+                    + straightLine + "\n");
+            break;
+        }
+    }
+
+    /**
+     * Output the corresponding customised String Index Out Of Bounds Exception message.
+     *
+     * @param keyword A keyword object to identify which message to output.
+     */
+    public void printInsufficientInfoException(Keywords keyword) {
+        switch (keyword) {
+            case delete:
```

Comment:
According to [Java coding standards](https://se-education.org/guides/conventions/java/intermediate.html), there should be no indentation for `case` clauses.

Reply:
Yep, fixed that already! Thank you for the tip!